framework/contracts/native/ica-client/src/queries.rs

```diff
@@ -10,8 +10,8 @@ pub const PACKET_LIFETIME: u64 = 60 * 60;
 
 pub fn config(deps: Deps, env: &Env) -> IcaClientResult<ConfigResponse> {
     Ok(ConfigResponse {
-        ans_host: AnsHost::new(deps.api, env)?.address,
-        registry_address: RegistryContract::new(deps.api, env)?.address,
+        ans_host_address: AnsHost::new(deps.api, env)?.address.to_string(),
+        registry_address: RegistryContract::new(deps.api, env)?.address.to_string(),
     })
 }
```

Comment:
@CyberHoward prefers to return `Addr` from queries if we have verified Address. Let's stick to one standard